scripts/ci/analyze-commits.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
@jelbourn In bash, this would require more work with the multiple lines. Since the bash output strips the line delimiters out. I don't say this won't work, but it would require more work and wouldn't be that elegant as in node.